### PR TITLE
OCNE Selector for UI

### PR DESCRIPTION
**Pull request by anders-swanson**

### CODE REVIEW

File: lib/shared/addon/components/cluster-driver/driver-ociocne/component.js [hunk 1]
```diff
@@ -270,25 +270,66 @@ export default Component.extend(ClusterDriver, {
 
     return out;
   }),
-  kubernetesVersionOptions: computed('config.region', 'primaryResource.cloudCredentialId', async function() {
+  ocneVersionOptions: computed('config.region', 'primaryResource.cloudCredentialId', async function() {
```
Comment (jmaron99) [inline lib/shared/addon/components/cluster-driver/driver-ociocne/component.js:273]: Some of these have an 'Options' suffix and some don't - is there a reason?

Comment (anders-swanson) [reply lib/shared/addon/components/cluster-driver/driver-ociocne/component.js:273]: I did not put `options` suffix for the tags. If you'd like, I can add this

Comment (jmaron99) [reply lib/shared/addon/components/cluster-driver/driver-ociocne/component.js:273]: No - was just curious

**Review by jmaron99 — APPROVED** [review]
Looks good to me but you may want to wait on the tigera vs calico naming preference?

**Comment by anders-swanson** [comment]
> Looks good to me but you may want to wait on the tigera vs calico naming preference?

Will follow up on this Monday. Tigera tag is going to be the commonly changed parameter here.

**Pull request merged**